packages/components/src/components/FormSelect/FormSelect.stories.tsx

```diff
@@ -0,0 +1,92 @@
+import type { Meta } from "@storybook/react";
+import React from "react";
+import { useUID } from "react-uid";
+import type { SelectProps } from "../Select";
+import { FormSelect } from "./FormSelect";
+
+const meta: Meta<typeof FormSelect> = {
+  title: "Components/Form/FormSelect",
```

Comment:
what about adding the subcomponents here? Or are we only doing it on some of the stories?

Reply:
This one really doesn't have subcomponents. Its just one that is a collection of components.